src/main/scala/org/broadinstitute/dsde/rawls/expressions/ExpressionParsing.scala

```diff
@@ -207,9 +207,13 @@ class ExpressionParser extends JavaTokenParsers {
     // Note that this requires traversing one additional time, from the "last" vertices to their subordinate attribute map vertices.
     FinalResult(
       lastVertices.map((v: Vertex) => {
-        v.getVertices(Direction.OUT, EdgeSchema.Own.toLabel("attributes")).headOption match {
-          case Some(mapVertex) => mapVertex.getProperty(attributeName).asInstanceOf[Object]
-          case None => throw new RuntimeException("Boo hoo, what should I do")
+        if (Attributable.reservedAttributeNames.contains(attributeName)) {
```

Comment:
This block was recently changed, so you should rebase and resolve any conflicts.